src/Compiler/Service/ServiceAnalysis.fs

```diff
@@ -451,3 +452,1091 @@ module UnusedDeclarations =
             let unusedRanges = getUnusedDeclarationRanges allSymbolUsesInFile isScriptFile
             return unusedRanges
         }
+
+module UnnecessaryParentheses =
+    open System
+
+#if !NET7_0_OR_GREATER
+    [<Sealed; AbstractClass; Extension>]
+    type ReadOnlySpanExtensions =
```

Comment:
Feel free to split those to a separate module/file.

Reply:
c96ec3a